internal/application/application.go

```diff
@@ -342,3 +365,42 @@ func (al ApplicationList) Swap(i, j int) {
 func (al ApplicationList) Less(i, j int) bool {
 	return al[i].Name < al[j].Name
 }
+
+// Unstage deletes either all PipelineRuns of the named application, or all but the current.
+func Unstage(app, org, stageIdCurrent string) error {
+	ctx := context.Background()
+
+	cluster, err := kubernetes.GetCluster()
+	if err != nil {
+		return err
+	}
+
+	cs, err := versioned.NewForConfig(cluster.RestConfig)
+	if err != nil {
+		return err
+	}
+
+	client := cs.TektonV1beta1().PipelineRuns(deployments.TektonStagingNamespace)
+
+	l, err := client.List(ctx, metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("app.kubernetes.io/name=%s,app.kubernetes.io/part-of=%s",
+			app, org),
+	})
+	if err != nil {
+		return err
+	}
+
+	for _, pr := range l.Items {
+		if stageIdCurrent != "" && stageIdCurrent == pr.ObjectMeta.Name {
+			fmt.Printf("skip\n")
```

Comment:
This looks like a leftover from debugging?

Reply:
Yes, d*t. Will remove.

Reply:
Done with 6e0d35f397412cbd7143acd3450e98600911dd05